scipy/optimize/tests/test_bracket.py

```diff
@@ -265,14 +285,16 @@ def test_input_validation(self):
         with pytest.raises(ValueError, match=message):
             _bracket_root(lambda x: x, -4, 4, maxiter="shrubbery")
 
-
-    def test_special_cases(self):
+    @pytest.mark.skip_xp_backends(np_only=True,
+                                  reason="scalar inputs use NumPy backend.")
+    def test_special_cases(self, xp):
```

Comment:
I think it's OK to adjust the scalar inputs to be arrays. I think that's what I did in the `nsum` conversion yesterday.